website/pages/docs/integrate/nia-integration.mdx

```diff
@@ -7,13 +7,13 @@ description: Guide to partnership integrations for Consul NIA
 
 # Network Infrastructure Automation Integration Program
 
-HashiCorp’s Network Infrastructure Integration (NIA) Program allows partners to build integrations that allow customers to automate dynamic application workflows leveraging network and security infrastructure at runtime. Detailed below is the approach to integrate your networking technology along with the clearly defined steps, links to information sources, and checkpoints.
+HashiCorp’s Network Infrastructure Automation (NIA) Integration Program allows partners to build integrations that allow customers to automate dynamic application workflows leveraging network and security infrastructure at runtime. Detailed below is the approach to integrate your networking technology along with the clearly defined steps, links to information sources, and checkpoints.
```

Comment:
This one tripped me up trying to trace back NIA to "Network Infrastructure Integration". Another possibly is to keep the original and omit "(NIA)" here?

Reply:
Good catch - I forgot the Automation portion. Thank you!